private_repo/ansible/inventory/dev/Core/hosts

```diff
@@ -197,6 +197,9 @@ yarn
 core
 local
 
+[ml-data-pipeline]
```

Comment:
Why do we need this in Core directory?

Reply:
By the next release, it'll be containerized and moved to core. If we keep this is DP now, we'll have to migrate it from there.